### DISCUSSION TITLE
blog/how-to-set-up-a-blockchain-network-with-hyperledger-besu

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#46] [Announcements]
# blog/how-to-set-up-a-blockchain-network-with-hyperledger-besu

If you are a Java developer interested in cryptocurrencies and blockchain-related technologies, you should probably have heard about Hyperledger Besu, the official Java client of the Ethereum blockchain. In this post, we’re going to set up a production-ready Ethereum client using Besu.

https://www.cesarsotovalero.net/blog/how-to-set-up-a-blockchain-network-with-hyperledger-besu.html

**Comment by Ivan-Dosev** [comment]
Hi, thank you for the article. Can you also describe how to add external node to private besu network. I mean like at the beginning where you describe the configuration file, is it same wat to add a node to private network. I have created 3 running nodes on my laptop and now I want to create a fourth on my second laptop and connect it to the first 3.
